app/src/main/res/values/dimen.xml

```diff
@@ -274,6 +274,9 @@
     <dimen name="settings_dialog_width">600dp</dimen>
     <dimen name="settings_dialog_height">360dp</dimen>
 
+    <!-- Language and voice settings -->
+    <dimen name="clear_history_height">400dp</dimen>
```

Comment:
Is setting a fixed height the right approach  ? We may have problems in the future if a SOP upgrade changes some if the UI styles. Cant we set the height to a dynamic value, so that it depends on the content ?

Reply:
I agree, fixed sizes are bad for interoperability among multiple devices. We probably need a scroll there...

Reply:
I'd rather avoid scroll if possible, usein content-size height for the window, with a max-height that would enable scroll as you said, but only in that case.

Reply:
Right that's what I meant